homeassistant/components/local_ip/sensor.py

```diff
@@ -25,10 +26,14 @@ class IPSensor(SensorEntity):
 
     _attr_unique_id = SENSOR
     _attr_icon = "mdi:ip"
+    _attr_has_entity_name = True
 
     def __init__(self, name: str) -> None:
         """Initialize the sensor."""
-        self._attr_name = name
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, DOMAIN)},
+            name=name,
```

Comment:
I am pondering if this should set that default name property instead (i think there is such a thing). I feel like there should be mor integrations that could share this device. It is representing the local host after all.

Reply:
I think it is not really correct. The name of the device isn't really "Local IP". Making this a device can be quite confusing.

Reply:
Should we make it a service instead? Seems also a bit odd.

Reply:
No, not a device or service at all. I think we can just let this one without a `DeviceInfo`.